### PR TITLE
Rename Scratch Namespace to Code

### PR DESCRIPTION
**Pull request by edfloreshz**

### CODE REVIEW

**Comment by tintou** [comment]
Can you remove the changes you've done with the .po and .pot files, those are made automatically after being merged to master.

**Comment by edfloreshz** [comment]
As requested, .po and .pot files are back to normal.

File: po/af.po [hunk 1]
```diff
@@ -736,8 +736,8 @@ msgstr ""
 #~ msgid "Text Editor"
 #~ msgstr "Teks Redigeerder"
 
-#~ msgid "About Scratch"
-#~ msgstr "Inligting oor Scratch"
+#~ msgid "About Code"
+#~ msgstr "Inligting oor Code"
```
Comment (danrabbit) [inline po/af.po]: Looks like this branch still contains changes to .po files

**Comment by edfloreshz** [comment]
I'll be back...

**Pull request closed without merging**